src/index.js

```diff
@@ -77,12 +77,14 @@ export default class CSSSplitWebpackPlugin {
     imports = false,
     filename = '[name]-[part].[ext]',
     preserve,
+    listenForEmit = false
```

Comment:
I think this name is a bit misleading? Maybe we can do like the last call plugin does and add `phase` or something like that which defaults to `optimize-chunk-assets`. Documentation for this should also be added to the README.

Reply:
@izaakschroeder Yea i agree.
Here, we are specifying compiler plugin, which is `emit` or `this-compilation` (as default). Currently only accept `emit` as an option. I've re-named as `compilerPhase`, please review.